install.sh

```diff
@@ -290,18 +285,20 @@ install_aur() {
   echo "Installing from the AUR."
   echo
 
-  prev_dir="$PWD"
   tmp_dir="$(mktemp -d)"
-  cd "$tmp_dir"
 
-  echo "+ Downloading PKGBUILD from https://aur.archlinux.org/cgit/aur.git/snapshot/code-server.tar.gz"
-  SKIP_ECHO=1 sh_c 'curl -fsSL https://aur.archlinux.org/cgit/aur.git/snapshot/code-server.tar.gz | tar -xz --strip-components 1'
-  unset SKIP_ECHO
-  echo "+ Downloaded into $tmp_dir"
-  sh_c makepkg -si
+  echo "+ Downloading PKGBUILD into $tmp_dir from https://aur.archlinux.org/cgit/aur.git/snapshot/code-server.tar.gz"
+  curl -fsSL https://aur.archlinux.org/cgit/aur.git/snapshot/code-server.tar.gz | tar -xzC "$tmp_dir"
```

Comment:
Should this still use `sh_c` so it doesn't download during a dry run?

Reply:
Has to to get the $VERSION on the AUR.

Reply:
Ah yeah that makes sense. We could set a dummy version but tbh this seems fine to me.

Reply:
Better to have the real version even in a dry run for sure.

Reply:
Yea, otherwise you don't know what's being installed.